package.json

```diff
@@ -168,7 +168,7 @@
   "dependencies": {
     "core-js": "^3.26.0",
     "fflate": "^0.7.4",
-    "rrweb": "2.0.0-alpha.11",
+    "rrweb": "^2.0.0-alpha.12",
```

Comment:
Having the carat in the version worries me since I think we need to be in sync with the UI, right? Maybe I am wrong.

Reply:
fair point, should probably be more explicit